app/assets/stylesheets/notifications-welcome-broadcast.scss

```
@@ -0,0 +1,20 @@
+// TODO[team-delightful]: Refactor nesting in this file
```

```suggestion
// TODO: [@thepracticaldev/delightful]: Refactor nesting in this file
```
I often grep for `@thepracticaldev/delightful` since that's our name in the codeowner's file, so this will keep consistent with that format!

---

Changed!